src/main/scala/org/constellation/util/Simulation.scala

```diff
@@ -410,6 +418,13 @@ object Simulation {
 
     assert(awaitCheckpointsAccepted(apis, numAccepted = 3))
 
+    true
+
+  }
+}
+
+/*
```

Comment:
Why do we comment on this code? 
why we don't remove it?

Reply:
I've commented everything after the place where the cluster is triggered to work

Reply:
Does it affect e2e as well ?

Reply:
If we need this code it shouldn't be commented 
If we don't need this code it should be removed (git remember if we need it)

Reply:
I'll prepare a separate test then